src/rez/cli/env.py

```diff
@@ -29,8 +29,11 @@ def setup_parser(parser, completions=False):
         help="skip loading of startup scripts")
     command_action = parser.add_argument(
         "-c", "--command", type=str,
-        help="read commands from string. Alternatively, list command arguments "
-        "after a '--'")
+        help="execute command within rez environment and exit, instead of "
+        "starting an interactive shell. Alternatively, list command after a "
+        "'--'. The command and arguments passed to '-c' must be passed in as "
```

Comment:
```suggestion
        "starting an interactive shell. Alternatively, list command after a "
        "'--'. The command and arguments passed to '-c' must be passed in as "
```
Should this  (`--`) be documented in the program's help (the module docstring) instead of in the `-c` argument? One thing I would like (and I think I made an issue for this) is to improve our CLI docs. For example, we could show possible combinations like in https://pip.pypa.io/en/stable/cli/pip_install/.

Reply:
I like the showing possible combinations idea, since `rez-env` effectively has several different call signatures and uses.

The CLI doesn't currently use it, but the [`ArgumentParser.usage`](https://docs.python.org/3/library/argparse.html#usage) attribute is probably how the multiple usages (and the `--` mode) should be documented. Right now, the usage is auto-generated from the argument list. It's pretty unruly and redundant; we don't need all the args listed in the example usage. 

Maybe save that for a separate ticket though, since we would need to change the parser setup to include custom `usage` information for each subparser, then we can move some of this information there.

Reply:
Sounds good.